modules/social_features/social_event/modules/social_event_an_enroll/src/Form/EventAnEnrollActionForm.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Drupal\social_event_an_enroll\Form;
+
+use Drupal\social_event\Form\EnrollActionForm;
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\node\Entity\Node;
+
+/**
+ * Class EventAnEnrollActionForm.
+ *
+ * @package Drupal\social_event_an_enroll\Form
+ */
+class EventAnEnrollActionForm extends EnrollActionForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'event_an_enroll_cancel_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $nid = $this->routeMatch->getRawParameter('node');
+    $token = $this->getRequest()->query->get('token');
+
+    // Load node object.
+    if (!is_null($nid) && !is_object($nid)) {
+      $node = Node::load($nid);
+    }
+
+    if (!empty($token) && social_event_an_enroll_token_exists($token, $nid)) {
+      $form['event'] = [
+        '#type' => 'hidden',
+        '#value' => $nid,
+      ];
+
+      $form['enroll_for_this_event'] = [
+        '#type' => 'submit',
+        '#value' => $this->t('Enrolled'),
+        '#attributes' => [
+          'class' => [
+            'btn',
+            'btn-accent brand-bg-accent',
+            'btn-lg btn-raised',
```

Comment:
We should keep one class per line so that it's more easily manipulated in other places (e.g. using attributes.removeClass(classname)).